WordPressShared/WordPressShared/Core/Views/WPStyleGuide+DynamicType.swift

```diff
@@ -0,0 +1,182 @@
+/// Extension on WPStyleGuide to use Dynamic Type fonts.
+///
+extension WPStyleGuide {
+    static let defaultTableViewRowHeight: CGFloat = 44.0
+
+    static let notoLoaded = { () -> Bool in
+        WPFontManager.loadNotoFontFamily()
+        return true
+    }()
+
+    /// Configures a table to automatically resize its rows according to their content.
+    ///
+    /// - Parameters:
+    ///     - tableView: The tableView to configure.
+    ///
+    public class func configureAutomaticHeightRowsForTableView(_ tableView: UITableView) {
```

Comment:
Let's give this a more swifty method name. What do you think of `configureAutomaticHeightRows(for tableView: UITableView)`?